### PR TITLE
Update react-ci-cd.yaml

### PR DESCRIPTION
**Pull request by adam-taitano**
Attempt to fix gh-pages deployment.

### CODE REVIEW

**Comment by adam-taitano** [comment]
I am hopeful this little addition will cause it to deploy correctly to our github pages url. Right now the readme.md is showing up at https://taylorknoah.github.io/FullStack-GroupProject-Spring22/ and the only difference I notice between my test repo and our real one is that this one currently downloads an additional Jekyll artifact upon deployment. This Jekyll artifact seems to offset our production-files build, maybe because it's used to create a blank page. 
So, by forcing it to publish based on the latest build, hopefully it will find our latest build and won't have to use the Jekyll process.

TL;DR - Maybe adding 'force_orphan: true' will fix our gh-pages deployment ^_^

**Review by TaylorKNoah — APPROVED** [review]
